### PR TITLE
feat: updates to avatar selection components

### PR DESCRIPTION
**Pull request by nickreynolds**

### CODE REVIEW

File: packages/components/src/Account/Auth/UserSetAvatar.tsx [hunk 1]
```diff
@@ -135,12 +143,14 @@ export class UserSetAvatar extends React.Component<UserSetAvatarAuthProps, UserS
...
                       onClick={() => this.setState({ image: undefined })}
+                      disabled={this.state.saveInProgress}
                     >
                       Go back
                     </InvertedButton>
                     <SaveButtonContainer>
-                      <Button size={buttonSizes.SMALL} textTransform={"none"} type={"submit"}>
-                        Save
+                      <Button size={buttonSizes.SMALL} textTransform={"none"} type={"submit"} disabled={this.state.saveInProgress}>
+                        {this.state.saveInProgress && <>Saving <ClipLoader size={16} /></>}
```
Comment (tobek) [inline packages/components/src/Account/Auth/UserSetAvatar.tsx]: Low priority but FYI I recently added a `loading` prop to `Button` that, when true, replaces its children with `ClipLoader`, for uses like this.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.27 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**